### ISSUE TITLE
Add es-check to make sure I'm transpiling correctly

### ISSUE THREAD

**Issue by sebnitu** [sebnitu/vrembem#347]
## Problem

I've been reading a lot about bundlers and their default behavior with ES6 and module imports. It'd be awesome to incorporate some kind of check to make sure my bundles are being output correctly.

## Possible Solution

After reading this article: [Lessons Learned From a Year of Fighting With Webpack and Babel](https://levelup.gitconnected.com/lessons-learned-from-a-year-of-fighting-with-webpack-and-babel-ce3b4b634c46), I got the suggestion to use [es-check](https://github.com/dollarshaveclub/es-check) for making sure I'm outputting the correct versions in my bundles.

__Additional resources:__

- [How to handle npm modules which include es6](https://github.com/babel/babel-loader/issues/171)

**Comment by sebnitu** [comment]
This is no longer needed after #394
